### ISSUE TITLE
Geth Bad Blocks Error

### ISSUE THREAD

**Issue by joeykrug** [ethereum/go-ethereum#1292]
I was on a private chain and one of the nodes got stuck and kept getting this error:

```0618 13:36:39.085722    6775 chain_manager.go:801] Bad block #25326 (d4d024e532c47afa8257d70e9d94145316a02e4866f515adf460fb43467256a9)
I0618 13:36:39.085772    6775 chain_manager.go:802] invalid merkle root. received=7d7d803dd4af4609b34ee6449c32fea46f2ad38e21912c116c734d67fb92ef9e got=c9c32232f34874cf666548c41c1fc780011033022c8645c94ce49c8ddfd85bea
I0618 13:36:39.086020    6775 downloader.go:224] Synchronisation failed: block downloading canceled (requested)```

I even tried removing all its peers and connecting it to just my node (which had all proper blocks) and it still kept giving that error whilst trying to sync after being off a few hours.  

l eventually just nuked its chain and let it start syncing from scratch and it worked.

**Comment by obscuren** [comment]
This is on `develop`?

**Comment by obscuren** [comment]
Did you have a crash prior to this? Did you `killall geth`?

**Comment by joeykrug** [comment]
Yes on develop --- no crash that I know of.  Was loading the Augur demo page when I noticed that it was behind on blocks, logged into the geth box that it connects to and i noticed it was just stuck on Bad block #25326 and then it cancelled sync, then tried again, got the same error and just repeated.  I then tried killing geth, restarting and had the same issue.  Finally just nuked the chain and all was well.
